_data/en/store.yml

```
@@ -137,6 +137,9 @@ items:
     eon-timer:
         title: Eon Timer
         intro: Time tracking for Mac, seamlessly integrated with Kimai – both On-Premise and Cloud.
+    playmoweb-kimtrack:
+        title: KimTrack
+        intro: Native desktop client for macOS, Windows, and Linux to track time from your menu bar.
```

```suggestion
        intro: Native desktop client for macOS, Windows, and Linux to track time from your menu bar.
```
Is your app really "native" ? It feels like a  Native desktop client is partly misleading here (e.g. its not Swift on MacOS)

BTW: there is no About dialog. 
Please include one and clarify that KimTrack is NOT affiliated with me. 
And include support information, I get many support requests for third party plugins/apps.